trace/trace.go

```diff
@@ -23,6 +23,9 @@ import (
 	"runtime"
 	"sync"
 	"time"
+	"go.opencensus.io/stats"
```

Comment:
Could we separate non stdlib imports from stdlib imports?

Reply:
agh, I always forget about this! I configured GoLand to do it automatically so hopefully that will help.